exercises/concept/chessboard/.docs/introduction.md

```diff
@@ -0,0 +1,97 @@
+# Introduction
+
+In Go, you can iterate a [slice](slices) using `for` but you could also use
+`range`. Range allows you to also iterate through a map or a
+[channel](channels). While iterating over a [channel](channels) is out of the
+scope for this exercise, you will find useful to iterate over a [map](maps).
```

Comment:
I think it makes sense to mention this in the general concept introduction, but it makes less sense in the context of an exercise which doesn't involve channels. You could trim that part out.